pkg/models/user_auth.go

```diff
@@ -84,6 +85,7 @@ type LoginUserQuery struct {
 	User       *User
 	IpAddress  string
 	AuthModule string
+	Cfg        *setting.Cfg
```

Comment:
@sakjur I don't see the point in the hassle. Should we make a new type just to filter down the configuration used? That would cause a lot of extra work if we were going to follow the principle over the whole codebase.

In any case, we're dropping the bus pattern (at least I was told when I joined), so it will go away in favour of something cleaner.

Reply:
I'm not convinced by my own opinion's strength here, so I'll mark this as resolved.